src/OrbitGl/MemoryTrack.h

```diff
@@ -31,7 +31,7 @@ class MemoryTrack : public GraphTrack<Dimension>, public AnnotationTrack {
                               module_manager, capture_data),
         AnnotationTrack() {
     // Memory tracks are collapsed by default.
-    this->collapse_toggle_->SetCollapsed(true);
+    this->header_->GetCollapseToggle()->SetCollapsed(true);
```

Comment:
As above. But as this code re-appears now quite often, I would really like to see it encapsulated.